app/views/layouts/_auth_bearer_token.html.haml

```diff
@@ -6,9 +6,18 @@
 - url      = url_for(:action => change_url, :id => "#{rec_id}")
 - url_json = {:interval => '.5', :url => url}.to_json
 - token_label ||= _("Token")
+- uid_label ||= _("Username")
```

Comment:
@oschreib let's use OpenShift nomenclature, I think it should be "Service Account Name" or simply "Account Name". Any suggestion here @deads2k @smarterclayton ? The token would belong most of the time to a service account but it could be not limited to that (I suppose, correct me if I am wrong), so do you have any preference for how we call the "username" field?